src/shared-components/DataGridWrapper/DataGridWrapper.spec.tsx

```
@@ -241,6 +398,176 @@ describe('DataGridWrapper', () => {
+        searchConfig={{
+          enabled: true,
+          fields: ['name', 'age'],
+        }}
+      />,
+    );
+
+    expect(screen.getByRole('textbox')).toBeInTheDocument();
+  });
+
+  test('handles action column rendering', () => {
+    const actionColumn = (row: TestRow) => <button>Edit {row.name}</button>;
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)